From 52607780c02ee90a93bd40e91c5263e55e7a351c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Wed, 5 Oct 2016 15:54:02 +0200
Subject: [PATCH] Adapt a test to libpcap-1.8.0
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

libpcap-1.8.0 changed error message on invalid filter expression from
"syntax error" to "syntax error in filter expression: syntax error".

This patch adjust the t/09-error.t for that.

CPAN RT#117831

Signed-off-by: Petr Písař <ppisar@redhat.com>
---
 t/09-error.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/09-error.t b/t/09-error.t
index bcd2a13..ef8f0b2 100644
--- a/t/09-error.t
+++ b/t/09-error.t
@@ -22,7 +22,7 @@ is(   $@,   '', "compile() with an invalid filter string" );
 is(   $res, -1, " - result must not be null: $res" );
 eval { $err = Net::Pcap::geterr($pcap) };
 is(   $@,   '', "geterr()" );
-like( $err, '/^(?:parse|syntax) error$/', " - \$err must not be null: $err" );
+like( $err, '/(?:parse|syntax) error$/', " - \$err must not be null: $err" );
 
 # Testing compile() with a valid filter
 eval { $res = Net::Pcap::compile($pcap, \$filter, "tcp", 0, $mask) };
-- 
2.7.4

